packages/repository/src/relations/has-many/has-many-through.repository.ts

```diff
@@ -98,3 +100,75 @@ export interface HasManyThroughRepository<
     },
   ): Promise<void>;
 }
+
+export class DefaultHasManyThroughRepository<
+  TargetEntity extends Entity,
+  TargetID,
+  TargetRepository extends EntityCrudRepository<TargetEntity, TargetID>,
+  ThroughEntity extends Entity,
+  ThroughID,
+  ThroughRepository extends EntityCrudRepository<ThroughEntity, ThroughID>,
+  > implements HasManyThroughRepository<TargetEntity, TargetID, ThroughEntity> {
+
+  constructor(
+    public targetRepositoryGetter: Getter<TargetRepository>,
```

Comment:
Should the constructor pass in a target/through constraint like the original reference PR (for example: https://github.com/strongloop/loopback-next/pull/2359/files#diff-a35f109a989bf5b2ac1206db18756a0cR63) or is it intentionally omitted to avoid implementation details in the initial PR?

Reply:
Thanks for reviewing. I tried different solutions if there is another way to get the constraint. But giving up for now and copied from the referenced PR.